x-pack/plugins/watcher/server/models/action_status_model/action_status_model.ts

```diff
@@ -62,7 +62,7 @@ export const buildClientActionStatusModel = (serverActionStatusModel: ServerActi
     lastSuccessfulExecution,
   } = serverActionStatusModel;
   const state = deriveState(serverActionStatusModel);
-  const isAckable = deriveIsAckable(state);
+  const isAckable = serverActionStatusModel.actionStatusJson.ack.state === 'ackable';
```

Comment:
This change means that we no longer needed the tests about the relationship between `isAckable` and `state`, above, because there is no longer any relationship.